onnx_tf/handlers/backend/cumsum.py

```diff
@@ -1,35 +1,32 @@
 import tensorflow as tf
 
-from onnx_tf.common import exception
 from onnx_tf.handlers.backend_handler import BackendHandler
 from onnx_tf.handlers.handler import onnx_op
-from onnx_tf.handlers.handler import partial_support
-from onnx_tf.handlers.handler import ps_description
 from onnx_tf.handlers.handler import tf_func
+from onnx_tf.common import sys_config
 
 
 @onnx_op("CumSum")
 @tf_func(tf.math.cumsum)
-@partial_support(True)
-@ps_description(
-    "CumSum inputs in uint32/uint64 " + "are not supported in Tensorflow."
-)
 class CumSum(BackendHandler):
-  @classmethod
-  def args_check(cls, node, **kwargs):
-    supported_dtype = [
-        tf.bfloat16, tf.half, tf.float32, tf.float64, tf.uint8, tf.uint16,
-        tf.int8, tf.int16, tf.int32, tf.int64, tf.complex64, tf.complex128
-    ]
-    x = kwargs["tensor_dict"][node.inputs[0]]
-    if x.dtype not in supported_dtype:
-      exception.OP_UNSUPPORTED_EXCEPT(
-          "CumSum input in " + str(x.dtype) + " which", "Tensorflow")
 
   @classmethod
   def version_11(cls, node, **kwargs):
+
+    def dtype_cast(x):
+      cast_map = {tf.uint32: tf.int64}
+      if sys_config.auto_cast:
+        cast_map[tf.uint64] = tf.int64
+      x = tf.cast(x, cast_map[x.dtype]) if x.dtype in cast_map else x
+      return x
+
     tensor_dict = kwargs["tensor_dict"]
     x = tensor_dict[node.inputs[0]]
+
+    # handle data types that are not natively supported by Tensorflow
+    dtype = x.dtype
+    need_cast = dtype in [tf.uint32, tf.uint64]
+    x = dtype_cast(x) if need_cast else x
```

Comment:
@chinhuang007 in the case that user set auto_cast=False and x.dtype=tf.uint64, then this cumsum op will still send to TF and TF will throw exception about this unsupported datatype. 
How about throw this exception in args_check and stop the conversion, instead of waiting for TF to throw this exception?

Reply:
Thanks for the comments. I added args_check for both equal and cumsum so exception will be thrown from our code. Please review again.